mypy.ini

```diff
@@ -15,8 +15,8 @@ disallow_untyped_defs = true
 disallow_incomplete_defs = true
 show_error_codes = false
```

Comment:
Note that `show_error_codes = false` also exists here as legacy, and should be fixed eventually.

You can move it below the `# TODO` comment you have added.